eng/ci/templates/public/jobs/test-linux-public.yml

```diff
@@ -0,0 +1,29 @@
+jobs:
```

Comment:
I don't see any difference between the test-linux.yml and this file. We might not need to have two files with the same content - unless you think these two files might diverge in the future.

Reply:
They are using different agent pool. Only there is no change in Mac that's why it remains same for both.